packages/visualizations/addon/models/table.js

```diff
@@ -72,12 +72,14 @@ export default VisualizationBase.extend(Validations, {
       let category = get(metric, 'metric.category'),
           isTrend = ~(category.toLowerCase().indexOf('trend')),
           type = isTrend ? 'threshold' : 'metric',
-          longName = get(metric, 'metric.longName');
+          longName = get(metric, 'metric.longName'),
+          displayName = metricFormat(metric, longName);
 
       return {
-        field: metric.toJSON(),
         type,
-        displayName: metricFormat(metric, longName)
+        displayName,
+        placeholder: displayName,
```

Comment:
Why do we need `placeholder` and `displayName`?

Reply:
placeholder is used when displayName is empty

Reply:
If displayName is empty wouldn't that make placeholder also empty?

Reply:
Sorry for the confusion. I mean that if user empty displayName